Makefile

```diff
@@ -18,6 +18,10 @@ test-coverage:
 	       test_types.py \
 	    && coverage report -m
 
+.PHONY: man
+man:
+	cd man && ./build.py
+
```

Comment:
The [argparse-manpage docs](https://pypi.org/project/argparse-manpage/#:~:text=a%20global%0Avariable.-,use%20with%20setup.py,-First%2C%20you%20need) document a way to set up building manpages automatically as part of `setup.py`. I think that would preferable over having them in the repo.

If you want to track them in the repo, let's add a pre-commit hook which makes sure that the manpage is up-to-date:

```yaml
- repo: local
  hooks:
  - id: generate-manpage
    name: Generate manpage
    language: python
    additional_dependencies:
      - argparse_manpage
      - websocket-client
    entry: ./build_manpage.py
```

Reply:
I don't really want to generate the manpage on the fly because we're not doing that for any of the other manpages in the project either, but I like the idea of tracking whether the current one is outdated. Will do.